projects/cm/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cm",
```

Comment:
Matching same naming convention than the other Cotecna npm package, could we call the project similar to one the following suggestions?:

@cotecna/material
@cotecna-material
@cotecna/material-library

![image](https://user-images.githubusercontent.com/55638917/144221439-927baf38-f608-4d53-b02e-327942ad7fbe.png)

Reply:
Done 
![image](https://user-images.githubusercontent.com/15807748/144236499-3f921059-8a07-4bda-a45d-959622831e07.png)